ts/parts-3d/SVGRenderer.ts

```diff
@@ -715,13 +742,15 @@ H.SVGRenderer.prototype.cuboidPath = function (
         path1: Array<number>,
         path2: Array<number>
     ): Array<number|Array<number>> {
-        var ret = [[] as any, -1];
+        var ret = [[] as any, -1],
+            pathSide1: Array<number> = path1.map(mapSidePath) as any,
+            pathSide2: Array<number> = path2.map(mapSidePath) as any;
 
         path1 = path1.map(mapPath) as any;
         path2 = path2.map(mapPath) as any;
-        if (H.shapeArea(path1 as any) < 0) {
+        if (H.shapeArea(pathSide1 as any) < 0) {
```

Comment:
Quite a lot these castings for `pathSideX` variables - above first are defined as  `Array<number>` then casted to `any` and now here used as `any` again?

Reply:
It was needed to refactor and comment the whole method to make it more clear for everyone. I have additionally removed all castings in the method. Related commit: 78f4008